ragna/deploy/_ui/styles.py

```diff
@@ -159,133 +90,42 @@ def stylesheets(
 WELCOME_MODAL_WIDTH = 530
 
 
-APP_RAW = """
-
+CSS_VARS = """
 :root {
     --body-font: 'Inter', sans-serif !important;
     --accent-color: {{MAIN_COLOR}} !important;
 }
-
-* {
-    font-family: 'Inter', sans-serif;
-}
-
-.main {
-    padding: 0px !important;
-
-}
-
-.pn-wrapper {
-    padding: 0px;
-}
-
-div.card-margin {
-    margin: 0px !important;
-    height: 100% !important;
-}
-
-
-/* Hide the whole header */
-
-#header {
-    display: none;
-}
-
-
-div#content {
-    height: calc(100vh);
-}
-
-/* Fix the size of the modal */
-#pn-Modal {
-    --dialog-width: 800px !important;
-    --dialog-height:500px !important; 
-}
-
-/* Hide the default close button of the modal */
-.pn-modal-close {
-    display: none !important;
-}
-
-
-/* Hide the fullscreen button of the template */ 
-span.fullscreen-button {
-    display:none;
-}
-
 """.replace("{{MAIN_COLOR}}", MAIN_COLOR)
 
 
-CHAT_INTERFACE_CUSTOM_BUTTON = """
-:host(.solid) .bk-btn.bk-btn-default {
-    background-color: transparent;
-    color: gray;
-}
-
-
-.bk-btn {
-    border-radius: 0;
-    padding: 0;
-    font-size: 14px;
-}
-    """
-
-BK_INPUT_GRAY_BORDER = (
-    """ .bk-input {border-color: var(--neutral-color) !important;} """
-)
-
-
-SS_LABEL_STYLE = """
-:host {
-    margin-top:20px;
-}
-
-label, .bk-slider-title {
-    font-weight: bold;
-    margin-bottom: 5px;
-}
-
-.bk-slider-value {
-    font-weight: normal;
-}
-
-.noUi-target {
-    border-color: var(--accent-fill-active);
-}
-
-.noUi-handle {
-    border-radius: 50%;
-}
-
-.noUi-handle, .noUi-connects {
-    background-color: var(--clear-button-active);
-}
-
-.noUi-target[disabled='true'] {
-    border-color: darkgray;
-}
-
-
-.noUi-target[disabled='true'] .noUi-connect {
-    background-color: darkgray !important;
-}
+"""
+CSS and UI Helpers
+"""
 
 
-:host(.disabled) div.bk-slider-title {
-    color: darkgray !important;
-}
-
+def divider():
+    return pn.layout.Divider(styles={"padding": "0em 1em"})
 
-"""
 
+def css(
```

Comment:
Please apply the refactor suggestion from https://github.com/Quansight/ragna/pull/312#pullrequestreview-1867178050.

Reply:
Done